lib/config/primo.cfg

```diff
@@ -15,7 +15,7 @@ OPTIONS		 -O2 -SO2 -iquote. -Ca-IXIY -Cl-IXIY -DZ80 -DPRIMO -D__PRIMO__ -M -clib
 CLIB		default -lprimo_clib -lndos -LDESTDIR/lib/clibs/ixiy
 
 SUBTYPE     none 
-SUBTYPE     default -Cz+mamepp
+SUBTYPE     default -Cz+primo
```

Comment:
Can you add back:

`SUBTYPE mamepp -Cz+mamepp` to still allow creation of the snapshot files.

Reply:
I can fix it up, but you've created two subtypes named "default"!